chore(docs): Update Gatsby framework version support information (#33825)
gatsbyjs/gatsby: docs/docs/reference/release-notes/gatsby-version-support.md

---
title: Gatsby Framework Version Support
---

## Introduction

> This page illustrates the support plans for the major versionf of Gastby. This allows you to effectively plan their Gatsby framework upgrades.

Reviewer: ```suggestion
This page illustrates the support plans for the major versions of Gatsby. This allows you to effectively plan your Gatsby framework upgrades.
```

Generally, you can expect **1 major version per calendar year**. Although, there may be more than 1 major per year when a necessary evolution of Gatsby requires significant breaking change.

## Release and support schedule

Note: Future time ranges are listed when a specific target date is not yet determined.

| Version | Status                        | As Of            | Until           |
| ------- | ----------------------------- | ---------------- | --------------- |
| 4       | Active                        | October 21, 2021 | Summer 2022     |
| 3       | Maintenance Long-term support | October 21, 2021 | Summer 2022     |
| 2       | Maintenance Long-term support | March 3, 2021    | January 1, 2022 |

## Terminology

> - **Critical Patch** - A critical patche addresses an issue that prevents the Gatsby framework from delivering intended user value under typical conditions. Other items, such as minor bug fixes and new features are not considered "critical patches" to Gatsby.

Reviewer: ```suggestion
- **Critical Patch** - A critical patch addresses an issue that prevents the Gatsby framework from delivering intended user value under typical conditions. Other items, such as minor bug fixes and new features are not considered "critical patches" to Gatsby.
```
